site/plugins.json

```diff
@@ -407,7 +407,13 @@
     "version": "2.1.3",
     "compatibility": [
       {
-        "version": "2.1.3"
+        "version": "3.0.0",
+        "featureFlag": "use_lighthouse_version_3",
+        "nodeVersion": ">=16.15.1"
```

Comment:
According to https://github.com/netlify/plugins/blob/main/docs/CONTRIBUTING.md#update-a-plugin this isn't needed